src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -181,26 +193,49 @@ export default {
 			enableQFMobileVersion: false,
 			enableAlmostFundedRow: false,
 			HandOrangeIcon,
+			enableLoanRecommendations: false,
 		};
 	},
 	apollo: {
 		preFetch(config, client) {
 			return client.query({
-				query: experimentAssignmentQuery, variables: { id: THREE_LOANS_RECOMMENDED_ROW_EXP_KEY }
+				query: experimentAssignmentQuery,
+				variables: { id: THREE_LOANS_RECOMMENDED_ROW_EXP_KEY }
 			}).then(() => {
-				const recommendedLoansPromise = client.query({
-					query: flssLoansQueryExtended,
-					variables: prefetchedRecommendedLoansVariables
+				return client.query({
+					query: experimentAssignmentQuery,
+					variables: { id: LOAN_RECOMMENDATIONS_EXP_KEY }
 				});
-
-				return Promise.all([
-					client.query({ query: userInfoQuery }),
-					client.query({ query: experimentAssignmentQuery, variables: { id: FIVE_DOLLARS_NOTES_EXP } }),
-					client.query({ query: experimentAssignmentQuery, variables: { id: FLSS_ONGOING_EXP_KEY } }),
-					recommendedLoansPromise
-				]);
+			}).then(loanRecommendationsExp => {
+				const useRecommendations = loanRecommendationsExp?.data?.experiment?.version === 'b';
+
+				return client.query({ query: userInfoQuery })
```

Comment:
Actually since the result of the previous `experimentAssignmentQuery` call isn't needed for this query, all 3 queries can be ran in a single `Promise.all`

Reply:
done